addons/purchase_stock/tests/test_stockvaluation.py

```diff
@@ -33,6 +33,7 @@ def setUpClass(cls):
             # Ignore tax calculations for these tests.
             'supplier_taxes_id': False,
             'is_storable': True,
+            'categ_id': cls.env.ref('product.product_category_consumable').id,
```

Comment:
I may have missed something, but was the previous 'All' category configured in a specific way?
I feel like all the products that previously were assigned to that category should behave the same way, even without having any assigned category, so why do you assign categories to so many products in the tests who didn't have any special one before ?

Reply:
There are two reasons I had to add category on some test cases

1) Like here they update categories values of the product like change valuation, costing method in test cases that's why we need to specify category
[see line 61 in this file](https://github.com/odoo/odoo/pull/166323/files#diff-114a52640a90362788febee3f029aaf548119d5f719d99458b22f8bf7ec9766bR61)

2) In some test environment they didn't enable `stock_account.group_stock_accounting_automatic` group in test environment and if those test cases product has not category set then it'll set valuation of product to `manual_periodic` even though they change
- so either I add that group in test environment (I didn't do that because it thought adding group will impact other test case where they need `manual`) :see_no_evil: 
- or add category on product so it take valuation from category